references/classification/train.py

```diff
@@ -68,6 +82,14 @@ def _get_cache_path(filepath):
 
 
 def main(args):
+    if args.apex:
+        if (sys.version_info < (3, 0)):
+            print("Apex currently only supports Python 3. Aborting.")
+            return
+        if amp is None:
+            print("Please install apex from https://www.github.com/nvidia/apex to enable mixed-precision training.")
```

Comment:
Similarly here, we could raise a `RuntimeError`

Reply:
good suggestion. I've edited this.